js/src/components/google-mc-account-card/connect-mc/index.js

```diff
@@ -84,54 +101,69 @@ const ConnectMC = () => {
 		return (
 			<CreatingCard
 				retryAfter={ resultCreateAccount.error?.retry_after }
-				onRetry={ handleCreateAccount }
+				onRetry={ createAccount }
 			/>
 		);
 	}
 
+	const getIndicator = () => {
+		if ( ! hasFinishedResolution ) {
+			return <LoadingLabel />;
+		}
+
+		if ( isGoogleMCReady ) {
+			return <ConnectedIconLabel />;
+		}
+
+		if ( resultConnectMC.loading ) {
+			return (
+				<LoadingLabel
+					text={ __( 'Connecting…', 'google-listings-and-ads' ) }
+				/>
+			);
+		}
+
+		return (
+			<AppButton
+				isSecondary
+				eventName="gla_mc_account_connect_button_click"
+				eventProps={ { id: Number( value ) } }
+				onClick={ handleConnectMC }
+			>
+				{ __( 'Connect', 'google-listings-and-ads' ) }
+			</AppButton>
+		);
+	};
+
 	return (
 		<AccountCard
-			className="gla-connect-mc-card"
-			appearance={ APPEARANCE.GOOGLE_MERCHANT_CENTER }
-		>
-			<CardDivider />
-			<Section.Card.Body>
-				<Subsection.Title>
-					{ __(
-						'Connect to an existing account',
-						'google-listings-and-ads'
-					) }
-				</Subsection.Title>
-				<ContentButtonLayout>
-					<MerchantCenterSelectControl
-						value={ value }
-						onChange={ setValue }
-					/>
-					<AppButton
-						isSecondary
-						loading={ resultConnectMC.loading }
-						disabled={ ! value }
-						eventName="gla_mc_account_connect_button_click"
-						eventProps={ { id: Number( value ) } }
-						onClick={ handleConnectMC }
-					>
-						{ __( 'Connect', 'google-listings-and-ads' ) }
-					</AppButton>
-				</ContentButtonLayout>
-			</Section.Card.Body>
-			<Section.Card.Footer>
-				<CreateAccountButton
-					isLink
-					disabled={ resultConnectMC.loading }
-					onCreateAccount={ handleCreateAccount }
-				>
-					{ __(
-						'Or, create a new Merchant Center account',
-						'google-listings-and-ads'
-					) }
-				</CreateAccountButton>
-			</Section.Card.Footer>
-		</AccountCard>
+			className={ classnames( 'gla-connect-mc-card', className ) }
+			title={ __(
+				'Connect to existing Merchant Center account',
+				'google-listings-and-ads'
+			) }
+			helper={ __(
+				'Required to sync products so they show on Google.',
```

Comment:
I checked the designs and the wording is as per the designs:
https://www.figma.com/design/fqR0EHi63lWahRcVTKCcba/Google-Listings-%26-Ads-v2.x?node-id=7361-3088&node-type=frame&t=PAZXnb7jo194rJ9P-0

Reply:
The screenshot was also taken from the design document: https://www.figma.com/design/fqR0EHi63lWahRcVTKCcba/Google-Listings-%26-Ads-v2.x?node-id=7448-22449&node-type=frame&t=NwWu4XTpGolsJr8R-0

I knew the current wording existed in the design document as well. That's why I said there is a different wording rather than the wording is incorrect. The question is to confirm which one should be used.

Reply:
Asked for clarifications.